### ISSUE TITLE
topLabelComponent not show

### ISSUE THREAD

**Issue by onlystard** [Abhinandan-Kushwaha/react-native-gifted-charts#519]
i working with BarChart and custom topLabelComponent 
but code is Docs not working 

here is the code ex: 
```
import React from 'react';
import {View, Text} from 'react-native';
import {BarChart} from 'react-native-gifted-charts';

const App = () => {
    const data = [
        {value: 20, label: 'M'},
        {value: 30, label: 'T'},
        {
            value: 50,
            label: 'W',
            topLabelComponent: () => <Text style={{color: 'blue', fontSize: 18, marginBottom: 6}}>50</Text>,
        },
        {value: 40, label: 'T'},
        {value: 30, label: 'F'},
    ];
    return (
        <View style={{flex: 1, justifyContent: 'center', alignItems: 'center'}}>
            <BarChart width={300} data={data} frontColor='#177AD5' topColor={'red'} overflowTop={2} />
        </View>
    );
};

export default App;
```
I using "react-native": "0.73.2",
![Simulator Screenshot - iPhone 15 Pro - 2024-01-25 at 18 00 09](https://github.com/Abhinandan-Kushwaha/react-native-gifted-charts/assets/38348745/54f3c2d4-f9a7-43b3-8a73-2bbf75f164e0)

**Comment by Abhinandan-Kushwaha** [comment]
Thanks @onlystard for reporting this issue. I will release a new version with the fix today.

**Comment by Abhinandan-Kushwaha** [comment]
Hi @onlystard 
This has been fixed from version `1.4.3` onwards.🎉
Please use the latest version of _react-native-gifted-charts_.
